File: api/code-samples/javascript/governance/getStakingInfo.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const block = "latest";

    sdk.governance.getStakingInfo(block, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/governance/getStakingInfo.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockNumber = "latest"

sdk = OpenSDK(host)
governance_response = sdk.governance.get_staking_info(blockNumber)

print(json.loads(governance_response.response.data))
